tidb-in-kubernetes: add one faq item (#1609)
pingcap/docs: dev/tidb-in-kubernetes/tidb-operator-overview.md

TiDB Operator is an automatic operation system for TiDB clusters in Kubernetes. It provides a full management life-cycle for TiDB including deployment, upgrades, scaling, backup, fail-over, and configuration changes. With TiDB Operator, TiDB can run seamlessly in the Kubernetes clusters deployed on a public or private cloud.

> **Note:**
>

> > Only one TiDB Operator can be deployed in each Kubernetes cluster.

Reviewer: ```suggestion
> You can only deploy one TiDB Operator in a Kubernetes cluster.
```
